views/index.ejs

```diff
@@ -1,11 +1,101 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title><%= title %></title>
-    <link rel='stylesheet' href='/stylesheets/style.css' />
-  </head>
-  <body>
-    <h1><%= title %></h1>
-    <p>Welcome to <%= title %></p>
-  </body>
-</html>
+
+<head>
+  <link rel='stylesheet' href='/stylesheets/style.css' />
+</head>
+
+<body>
+  <header>
+    <% include layouts/header %>
+  </header>
+
+  <h1><%= title %></h1>
+
+  <div class="container mt-5">
+    <h4>Geo-Sentiment Location</h4>
+    <p>Enter a city's information to retrieve its geo-sentiment analysis:</p>
+    <div class="container justify-content-center align-items-center">
+      <form class="col-xs-8" method="POST" action="/mood">
+        <div class="form-group row">
+          <label class="col-sm-4 col-form-label">City:</label>
+          <div class="col-sm-5">
+            <input type="text" class="form-control" id="city" name="city" placeholder="City" required>
+          </div>
+        </div>
+        <div class="form-group row">
+          <label class="col-sm-4 col-form-label">State:</label>
+          <div class="col-sm-5">
+            <select select class="form-control" name="state" required>
+              <option value="" selected disabled>State</option>
+              <option value="Alabama">Alabama</option>
+              <option value="Alaska">Alaska</option>
+              <option value="Arizona">Arizona</option>
+              <option value="Arkansas">Arkansas</option>
+              <option value="California">California</option>
+              <option value="Colorado">Colorado</option>
+              <option value="Connecticut">Connecticut</option>
+              <option value="Delaware">Delaware</option>
+              <option value="District of Columbia">District of Columbia</option>
+              <option value="Florida">Florida</option>
+              <option value="Georgia">Georgia</option>
+              <option value="Guam">Guam</option>
+              <option value="Hawaii">Hawaii</option>
+              <option value="Idaho">Idaho</option>
```

Comment:
this hurts - any way we could load the options from the saved `cities.json`? That way we could have drop down menus for both options, then we wouldn't have to deal with free-text mismatches at all.  Not necessary for this PR but I think that would make using this a bit easier

Reply:
Future PR: move the coord retrieval client-side